analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.CSharp9.Part1.cs

```diff
@@ -11,6 +11,22 @@ public record SerializableRecord : ISerializable
         public virtual void GetObjectData(SerializationInfo info, StreamingContext context) { }
     }
 
+    [Serializable]
+    public record SerializableRecord_Positional(SerializableRecord PositionalSerializableField) : ISerializable
+    {
+        public SerializableRecord_Positional() : this((SerializableRecord)null) { }
+        protected SerializableRecord_Positional(SerializationInfo info, StreamingContext context) : this((SerializableRecord)null) { }
+        public virtual void GetObjectData(SerializationInfo info, StreamingContext context) { }
+    }
+
+    [Serializable]
+    public record SerializableRecord_PositionalNonserializable(NonSerializedType PositionalNonserializableField) : ISerializable    // FN, should raise similar rule not to put non-serializable parameters here. It cannot be annotated with [NonSerialized]
```

Comment:
I think the comment about the attributes is not valid. See the [spec](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/records#properties) and the sample:

https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQMwAIBOBTAxge02HQGEAKAbQDMBLbAG1HQDl8A7AZW0xoEN6aAL2zAAugFEAHrmwAHAC4126KQEp0Ab2QBfIA=